css/styles.css

```diff
@@ -18,7 +9,8 @@
 	min-height: 100px;
 	max-height: 260px;
 	border-radius: 0 0 3px 3px;
-	border: 1px solid rgb(238, 238, 238);
+	overflow: hidden;
+	overflow-y: auto;
```

Comment:
Moving this here from the `.notification-wrapper` hides the drop shadow as well as the little caret in Safari. Let me fix that.

Reply:
Fix is in #106